frontend/docs/docs/user-guide/running-crawl.md

```diff
@@ -34,6 +34,17 @@ Like exclusions, the number of [browser windows](workflow-setup.md#browser-windo
 
 Unlike exclusions, this change will not be applied to future workflow runs.
 
+## Pausing and Resuming Crawls
+
+If you need to pause your crawl at any point after it has started, whether to reassess, rescope, or for any other reason, you now have the option to do so.
```

Comment:
```suggestion
```

We generally try to be [as concise as possible](https://docs.browsertrix.com/develop/docs/#be-concise-avoid-you-statements) in docs and stick to actionable information.

Reply:
I somewhat disagree with this. Elsewhere in this same user guide we offer guidance on why someone may want to set exclusions, like

> While [exclusions](https://docs.browsertrix.com/user-guide/workflow-setup/#exclude-pages) can be set before running a crawl workflow, sometimes while crawling the crawler may find new parts of the site that weren't previously known about and shouldn't be crawled, or get stuck browsing parts of a website that automatically generate URLs known as ["crawler traps"](https://en.wikipedia.org/wiki/Spider_trap).

I think we should include a brief description of a pause use case with the "you now have", something like:

> If you need to reassess or rescope your crawl at any point after it has started, you can pause the running crawl.

Reply:
Added :)